index.html

```diff
@@ -1,27 +1,27 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="description" content="AntennaPod; the easy-to-use, flexible and open-source podcast manager and player for Android">
-    <meta name="keywords" lang="en" content="AntennaPod,Open-Source,Podcast,Manager,Player,Podcatcher,App,Android">
-    <link rel="shortcut icon" href="assets/img/favicon.ico">
-
-    <title>AntennaPod</title>
-
-    <!-- CSS -->
-    <link href="assets/bootstrap-3.3.7/css/bootstrap.min.css" rel="stylesheet">
-    <link href="assets/css/main.css" rel="stylesheet">
-	<link href='assets/fonts.css' rel='stylesheet' type='text/css'>
-	<link rel="stylesheet" href="assets/font-awesome-4.7.0/css/font-awesome.min.css">
-    
-    <!-- HTML5 for IE -->
-    <!--[if lt IE 9]>
-      <script src="assets/html5shiv-v3.7.0.js"></script>
-      <script src="assets/respond-v1.3.0.min.js"></script>
-    <![endif]-->
-  </head>
+<head>
+  <meta charset="utf-8">
```

Comment:
This indention is inconsistent with the rest of the file. Please either revert to how it was before or make it consistent with the rest of the file. To be honest, I don't really see a reason to change inconsistent indention to other inconsistent indention. The fewer changes to unrelated lines of code, the easier to review (and the more clean the git history).

Reply:
I made the headers consistent for all 3 files. The problem is that the source is not correctly intended, so it is impossible to do it right...

Reply:
I believe the point was that, just because it is impossible to do it right, it's better to leave as is (to keep the git history clean). Anyway, if the the comment about the new line in the list after a <br> tag is resolved, I think we can merge it @ByteHamster?